tests/test-coauthors-guest-authors.php

```diff
@@ -124,22 +124,15 @@ public function test_get_guest_author_thumbnail() {
 
 		$this->assertNull( $guest_author_obj->get_guest_author_thumbnail( $guest_author, 0 ) );
 
-		// Checks when guest author has thumbnail.
-		$filename = rand_str() . '.jpg';
-		$contents = rand_str();
-		$upload   = wp_upload_bits( $filename, null, $contents );
-
-		$this->assertTrue( empty( $upload['error'] ) );
-
-		$attachment_id = $this->_make_attachment( $upload );
+		$attachment_id = $this->factory->attachment->create_upload_object( __DIR__ . '/dummy-attachment.png' );
```

Comment:
I would suggest putting the image into `tests/fixtures/` directory, so that any files that are needed for tests, _but are not test files_ are kept separate.

Reply:
Thanks for pointing this out. The dummy image has been moved accordingly.